### PR TITLE
L1-287: Update the front end to inflation changes

### PR DESCRIPTION
**Pull request by youPickItUp**
Update inflation calculations:
* add type augmentation to type new API methods
* fetch inflation from backend and simplify frontend logic

### CODE REVIEW

**Comment by youPickItUp** [comment]
I've asked for review a bit too early. I still have to figure out how to add custom runtime methods to polkadot API - I'll probably copy azero.dev code :crossed_fingers:

**Comment by youPickItUp** [comment]
> I've asked for review a bit too early. I still have to figure out how to add custom runtime methods to polkadot API - I'll probably copy azero.dev code 🤞

Done :heavy_check_mark:

**Review by kostekIV — APPROVED** [review]
calculations look good

File: src/library/Hooks/useInflation/index.tsx [hunk 1]
```diff
@@ -1,70 +1,65 @@
...
+  }, [api]);
 
-    let stakedReturn = stakedFraction ? inflation / stakedFraction : 0;
-    stakedReturn *= 0.9;
+  return yearlyInflation;
+};
 
-    return {
-      idealInterest,
-      idealStake,
```
Comment (jalooc) [inline src/library/Hooks/useInflation/index.tsx:62]: I'm assuming these 2 are not used since you're not returning them from `calculateInflation()`?

**Review by jalooc — APPROVED** [review]
lgtm

File: src/library/Hooks/useInflation/index.tsx (hunk 1 above)
Comment (youPickItUp) [reply src/library/Hooks/useInflation/index.tsx:62]: yes :+1:

**Pull request merged**